xbmc/utils/PlayerUtils.h

```diff
@@ -8,10 +8,14 @@
 
 #pragma once
 
+#include <memory>
+
+class CApplicationPlayer;
 class CFileItem;
 
 class CPlayerUtils
 {
 public:
   static bool IsItemPlayable(const CFileItem& item);
+  static void advanceTempoStep(std::shared_ptr<CApplicationPlayer> appPlayer, bool increase);
```

Comment:
```suggestion
  static void AdvanceTempoStep(std::shared_ptr<CApplicationPlayer> appPlayer, bool increase);
```

According to code guidelines

Reply:
Also the bool would be better replaced by an enum class TempoStepMode for increased readability (TEMPO::UP, TEMPO::DOWN)  or something similar

Reply:
> Also the bool would be better replaced by an enum class TempoStepMode for increased readability (TEMPO::UP, TEMPO::DOWN) or something similar

very good idea, thanks